trulens_eval/tests/docs_notebooks/notebooks_to_test/all_tools.ipynb

```diff
@@ -1 +0,0 @@
-../../../../docs/trulens_eval/all_tools.ipynb
```

Comment:
basically, from my reading of the code yesterday, there was some script that:
1. Took a bunch of notebooks and combined them into one big notebook.
2. Essentially tried to convert the notebook into a script (so it stripped metadata and got rid of jupyer display specific stuff).
3. Ran this notebook as a test.

What's weird to me is that most of the notebooks in the big notebook were already running as separate tests and I can't imagine a good reason to put them all together. I asked about this in the Slack group but no one answered so I assume there's no good reason. There were two notebooks that weren't tested individually though, one of which I just added to the individual tests (i.e. `logging.ipynb`). The other `custom_feedback_functions.ipynb` is clearly not written in a way such that it can be run on its own and just happened to work as part of `all_tools.ipynb` presumably because a variable that isn't declared in the notebook itself happened to be declared in a previous notebook in `all_tools.ipynb`.